Processing_data/Classes/DataValue.cs

```diff
@@ -0,0 +1,12 @@
+namespace Processing_data.Classes
+{
+    /// <summary>
+    /// Represents data value from the JSON file
+    /// </summary>
+    public class DataValue
```

Comment:
this also may be a struct. 

When a struct has only value-typed fields, the equality operator compares the data bitwise (which is fast). This is one of the advantages of structs.

Reply:
updated this class
it is a struct now